Composer/packages/client/src/pages/home/Home.tsx

```diff
@@ -136,10 +132,6 @@ const Home: React.FC<RouteComponentProps> = () => {
     },
   ];
 
-  useEffect(() => {
-    fetchRecentProjects();
```

Comment:
move fetchRecentProjects to creationFlow to centralize resource fetch in sequence.